### PR TITLE
fix: fixed always yield true if statement

### LINKED ISSUE #1377: taskmanager: compare int with null
```
[WARNING] /Users/ark/Code/OpenMLDB/java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala:81: warning: comparing values of types Int and Null using `!=' will always yield true
[INFO]     if (TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS != null) {
```

### PR DESCRIPTION
**Pull request by wuteek**
* **Please check if the PR fulfills these requirements**

- The commit message follows our guidelines
- Tests for the changes have been added (for bug fixes / features)
- Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix

* **What is the current behavior?** (You can also link to an open issue here)

This statement always returns true
fixes taskmanager: compare int with null #1377

### CODE REVIEW

**Comment by wuteek** [comment]
I was wondering if I was able to get some help passing the coverage check? I am new to contributing

**Comment by lumianph** [comment]
> I was wondering if I was able to get some help passing the coverage check? I am new to contributing

No worry about that. It failed due to issues of some of our test cases (not related to your code).

File: java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala [hunk 1]
```diff
@@ -78,7 +78,7 @@ object SparkJobManager {
       launcher.setConf("spark.eventLog.dir", TaskManagerConfig.SPARK_EVENTLOG_DIR)
     }
 
-    if (TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS != null) {
+    if ((TaskManagerConfig.SPARK_YARN_MAXAPPATTEMPTS : java.lang.Integer) != null) {
```
Comment (vagetablechicken) [inline java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala]: `SPARK_YARN_MAXAPPATTEMPTS` is int, not Integer, no need to check if null

**Review by vagetablechicken — CHANGES_REQUESTED** [review]
You could do this:
```
if SPARK_YARN_MAXAPPATTEMPTS >= 1:
    setConf
```

**Comment by wuteek** [comment]
> You could do this:
> 
> ```
> if SPARK_YARN_MAXAPPATTEMPTS >= 1:
>     setConf
> ```

Hello, I pushed with the recommendations that you gave. Please let me know if any further issues occur. Thank you!

**Review by lumianph — APPROVED** [review]
LGTM

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**